src/HealthGPS.Console/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 find_package(fmt CONFIG REQUIRED)
 find_package(cxxopts CONFIG REQUIRED)
 find_package(nlohmann_json CONFIG REQUIRED)
+find_package(TBB CONFIG REQUIRED)
 find_path(RAPIDCSV_INCLUDE_DIRS "rapidcsv.h")
 find_package(Threads REQUIRED)
```

Comment:
Knowing nothing about CMake, I'm curious: where does CMake looks for all of these packages?

Reply:
Traditionally, CMake looks for these on your system and e.g. the TBB library will come with a CMake package config file. And traditionally this worked rather poorly on Windows :stuck_out_tongue:.

Health-GPS uses [vcpkg](https://vcpkg.io/) on top to manage dependencies so they are automatically built from source before everything else is (provided you have vcpkg installed!).